lib/client/commands/login.go

```diff
@@ -79,23 +81,23 @@ func (l *Login) Run(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	client, err := l.base.AuthClient(l.rng)
+	conn, err := l.base.Connect()
 	if err != nil {
 		return err
 	}
-
-	options := auth.LoginOptions{
-		Context: l.base.Context(),
-		MinWait: l.MinWaitTime,
-		Store:   l.base.Local,
-	}
-
-	token, err := client.Login(username, domain, options)
+	repeater := retry.New(retry.WithWait(l.MinWaitTime), retry.WithRng(l.rng))
```

Comment:
btw, two reasons I wanted to bring in the retry library:
1) So we'd have the fuzzying, WithFuzzy(1 second)
2) So we'd have the retry parameters exposed on the CLI like any other parameter.

not necessary to do for this PR.